### DISCUSSION TITLE
Open source Link on codespaces page doesn't work

### DISCUSSION THREAD

**Discussion by alex-rakowski** [community/community#60700] [Codespaces]
### Select Topic Area

Bug

### Body

The link on the [codespaces page](https://github.com/features/codespaces) for the open-source maintainers is returning a 404 page error. The link is at the bottom of the FAQ section, in the last dropdown option. 

Section: 
> Is Codespaces available for open source maintainers?
> Yes, we want open source maintainers and contributors to have a great experience with Codespaces. So, in addition to the free usage, we’re previewing free Prebuild Storage for open source organizations. Click [here](https://github.surveymonkey.com/r/2X56RCW) to apply.

Offending link:
https://github.surveymonkey.com/r/2X56RCW

**Comment by craiglpeters** [comment]
Thanks for pointing that out! I'll work on a solution to that.
